src/Symfony/Component/Cache/Adapter/DoctrineDbalAdapter.php

```diff
@@ -29,7 +29,7 @@
 
 class DoctrineDbalAdapter extends AbstractAdapter implements PruneableInterface
 {
-    protected $maxIdLength = 255;
+    protected int $maxIdLength = 255;
```

Comment:
Psalm has a point here: In `AbstractAdapterTrait` this property is declared nullable. Maybe we should stop overriding this property and move the assignment to the constructor?